src/radium-theme.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
   // Media Queries
   // --------
   mediaSizes: {
-    medium: "only screen and (min-width: 44em)"
+    medium: "only screen and (min-width: 44.063em)",
+    small: "only screen and (max-width: 44em)"
```

Comment:
This looks a little awkward... I presume `44em`–`44.062em` is a narrow edge case, but it would default to whatever styles aren’t in media queries.

How about a mobile-first approach here instead? The styles that aren’t in media queries would be viewports up to `44em` aka `small`, and anything in media queries would be for larger viewports and should override the default styles.